### PR TITLE
rename file for aws_ec2 resource

### PR DESCRIPTION
**Pull request by chris-rock**
This PR aligns the resource name with the file name

Signed-off-by: Christoph Hartmann <chris@lollyrock.com>

### CODE REVIEW

**Comment by chris-redekop** [comment]
Like IAM, EC2 is more of an AWS-service category (as opposed to a resource).  What about calling this resource `aws_ec2_instance`?  It would match our `aws_iam_X` resources and leave room for other EC2 resources (e.g. security groups, subnets, launch configurations, etc...).

We made these changes--and just these changes--on the issue82 branch.  If you'd like, we can merge those changes in now for this issue.  What do you think?

https://github.com/chef/inspec-aws/compare/issue82

**Comment by chris-rock** [comment]
@chris-redekop I renamed the resource to `aws_ec2_instance` as you proposed :-)

File: libraries/aws_ec2_instance.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # author: Christoph Hartmann
 
-class Ec2 < Inspec.resource(1)
-  name 'aws_ec2'
+class AwsEc2Instance < Inspec.resource(1)
+  name 'aws_ec2_instance'
   desc 'Verifies settings for an EC2 instance'
 
   example "
```
Comment (chris-redekop) [inline libraries/aws_ec2_instance.rb]: I like the rename, especially the deprecation. (Neat!)

I think the resource name should also be updated in the resource's examples, as well as in the README.

**Review by chris-redekop — CHANGES_REQUESTED** [review]
Looks good.  Just a request for doc updates.

File: libraries/aws_ec2_instance.rb (hunk 1 above)
Comment (chris-rock) [reply libraries/aws_ec2_instance.rb]: Thank you, I am going to improve that one.

**Review by chris-redekop — APPROVED** [review]
Looks good!

**Pull request merged**